README.md

```diff
@@ -104,6 +104,7 @@ The [User Guide](https://github.com/facebookincubator/create-react-app/blob/mast
 - [Updating to New Releases](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#updating-to-new-releases)
 - [Folder Structure](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#folder-structure)
 - [Available Scripts](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#available-scripts)
+- [Adding bundle analyzer script (source-map-explorer)](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-bundle-analyzer-scrip-source-map-explorer)
```

Comment:
I would move this closer to the end (perhaps before Deployment) and call the section Analyzing the Bundle Size